### ISSUE TITLE
Keep signs adjacent to numerals in output

### ISSUE THREAD

**Issue by dpeterson71** [r-lib/pillar#91]
The formatting of negative numbers is problematic with the negative sign sometimes many spaces away from the actual numbers.  Even though there may be red highlighting for the negative numbers, when the sign character is 4 or 5 spaces away from the number itself, it is not clear that the sign belongs as part of the number.

**Comment by krlmlr** [comment]
It feels easier to miss the negation sign if it's not aligned: with aligned negation signs you see at a glance which rows have negative values for that column. I agree that color helps more than the negation sign in many cases.

**Comment by dpeterson71** [comment]
I guess that is a matter of perception.  I have to completely disagree with that assessment.  With the current formatting, there are often cases where the sign is many spaces away and has   **ZERO** connection to the number itself.   The sign is an *integral* part of the number itself and should be connected to it.  When I see a number I don't want to have to look many places afar to figure out whether or not it is negative.  In fact, I have several cases where the formatting actually makes it look like the "negative" sign is a subtraction sign and the data look like a mathematical equation rather than numbers.  

In short, the current implementation of negatives actually *hinders* the understanding of the output rather than improving it.

**Comment by krlmlr** [comment]
It feels that the minus would be visible even worse if it was adjacent to the numbers. I agree that the minus e.g. in the `cpsTC` column looks like an arithmetic operator. We could easily replace the `"-"` with a `" -"` to make the connection more visible. How does that output look like?

**Comment by huftis** [comment]
FWIW, I find the current behaviour *extremely* confusing and hard to read. I don’t think adding a space before the `-` would help much. Having the `-` adjacent to the number, like we’re used to, works much better, IMHO.

**Comment by krlmlr** [comment]
Thanks for your feedback, the negation sign is printed adjacent to the number again.
